acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/signer/EthSignerProcessRunner.java

```diff
@@ -147,11 +147,22 @@ public void start(final String processName) {
       params.add(serverTlsOptions.getKeyStoreFile().toString());
       params.add("--tls-keystore-password-file");
       params.add(serverTlsOptions.getKeyStorePasswordFile().toString());
-      if (serverTlsOptions.getKnownClientsFile().isPresent()) {
-        params.add("--tls-known-clients-file");
-        params.add(serverTlsOptions.getKnownClientsFile().get().toString());
-      } else {
+      if (serverTlsOptions.getClientAuthConstraints().isEmpty()) {
         params.add("--tls-allow-any-client");
+      } else {
+        if (serverTlsOptions.getClientAuthConstraints().get().getKnownClientsFile().isPresent()) {
```

Comment:
nit: `serverTlsOptions.getClientAuthConstraints().get().getKnownClientsFile()` extracting it to a variable may help shorter if and params.add call.

Reply:
👍